src/main/java/com/grim3212/mc/pack/industry/block/BlockUraniumOre.java

```diff
@@ -22,46 +24,77 @@ public BlockUraniumOre() {
-				damage--;
+				//damage--;
+				damage = damage - damageFraction;
+				if (damage < 0) damage = 0;
 			}
 			if (((EntityPlayer) entityIn).inventory.armorInventory[2] != null && ((EntityPlayer) entityIn).inventory.armorInventory[2].getItem() == IndustryItems.anti_radiation_chest) {
-				damage--;
+				//damage--;
+				damage = damage - damageFraction;
+				if (damage < 0) damage = 0;
 			}
 			if (((EntityPlayer) entityIn).inventory.armorInventory[3] != null && ((EntityPlayer) entityIn).inventory.armorInventory[3].getItem() == IndustryItems.anti_radiation_helmet) {
-				damage--;
+				//damage--;
+				damage = damage - damageFraction;
+				if (damage < 0) damage = 0;
 			}
 			if (damage != 0) {
 				entityIn.attackEntityFrom(DamageSource.magic, damage);
 			}
 		} else {
 			if (damage != 0) {
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.0D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.25D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.5D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.75D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+				worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 2.0D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
+
 				entityIn.attackEntityFrom(DamageSource.magic, damage);
 			}
 		}
 	}
 
 	@Override
 	public void onBlockClicked(World worldIn, BlockPos pos, EntityPlayer playerIn) {
-		int damage = 4;
+
+		float damage = IndustryConfig.uraniumDamage;
+		float damageFraction = damage/4;
 		if (playerIn.inventory.armorInventory[3] != null && playerIn.inventory.armorInventory[3].getItem() == IndustryItems.anti_radiation_helmet) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (playerIn.inventory.armorInventory[2] != null && playerIn.inventory.armorInventory[2].getItem() == IndustryItems.anti_radiation_chest) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (playerIn.inventory.armorInventory[1] != null && playerIn.inventory.armorInventory[1].getItem() == IndustryItems.anti_radiation_legs) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (playerIn.inventory.armorInventory[0] != null && playerIn.inventory.armorInventory[0].getItem() == IndustryItems.anti_radiation_boots) {
-			damage--;
+			//damage--;
+			damage = damage - damageFraction;
+			if (damage < 0) damage = 0;
 		}
 		if (damage != 0) {
+			worldIn.spawnParticle(EnumParticleTypes.REDSTONE, pos.getX() + 0.5D, pos.getY() + 1.0D, pos.getZ() + 0.5D, 0.0D, 4.0D, 0.0D, new int[0]);
```

Comment:
Just playing to figure out how they work. Keep in mind all of this is new to me and this is actually the first thing I did. You can take them out or change them.